packages/shared/src/components/auth/OrDivider.tsx

```diff
@@ -3,22 +3,30 @@ import classNames from 'classnames';
 
 interface OrDividerProps {
   className?: string;
+  textColor?: string;
+  borderColor?: string;
```

Comment:
these probably best to be in an object
```suggestion
  classNames?: {
       generalClassName?: string;
       borderColor?: string;
       textColor?: string;
  };
  
```

but non blocking comment

Reply:
I thought about it, but would have to make all existing use-cases updated, which posed some concerns/issues we might miss so decided this was safer for this PR.

Reply:
yup all good, no worries